test/integration/gardenlet/controllerinstallation/controllerinstallation/controllerinstallation_suite_test.go

```diff
@@ -109,7 +109,9 @@ var _ = BeforeSuite(func() {
 	})
 
 	By("Create test client")
-	testClient, err = client.New(restConfig, client.Options{Scheme: kubernetes.GardenScheme})
+	testScheme := kubernetes.GardenScheme
+	Expect(resourcesv1alpha1.AddToScheme(testScheme)).To(Succeed())
```

Comment:
This mutates the global variable `kubernetes.GardenScheme` which is an anti-pattern in tests. 
Please create a new dedicated scheme, instead.

Reply:
This is the case with every other test. I could open a separate PR to streamline this. Is that fine?

Reply:
Sure thing 👍